doc/manual/source/store/derivation/outputs/input-address.md

```diff
@@ -6,26 +6,203 @@
 That is to say, an input-addressed output's store path is a function not of the output itself, but of the derivation that produced it.
 Even if two store paths have the same contents, if they are produced in different ways, and one is input-addressed, then they will have different store paths, and thus guaranteed to not be the same store object.
 
-<!---
+## Modulo fixed-output derivations {#hash-modulo}
 
-### Modulo fixed-output derivations
+So how do we compute the hash part of the output paths of an input-addressed derivation?
+This is done by the function `hashDerivationModulo`, shown below.
```

Comment:
Modulo is a binary operation. So hash derivation modulo _what_?
I'd say it's short for "modulo content addressed inputs" or some similar phrase.

Reply:
"modulo some transitive inputs" is vague put perhaps getting closer to the heart of the matter